### DISCUSSION TITLE
Π2018084 - Παραδοτέο 3

### DISCUSSION THREAD

**Discussion by Icelandan** [courses-ionio/sw#1348] [Show and tell]
Για το 3ο παραδοτέο έγινε χρήση Slack και mqttwarn.

**Άσκηση:** Γραμμή εντολών (no systemd)
**Κατηγορία:** Software
**Περιγραφή:** Create notifications on your server; Send notifications on important server events
**Asciinema link(s):** https://asciinema.org/a/yr0TwXAVpHlZiaDiJ57o2gUv0
https://asciinema.org/a/QJXzvz2BmplSEp0Lg69Xt7W1j
**Image Link:** https://drive.google.com/file/d/1bDnXf8oSDIF3KP2mMaN9LdYFPJqSv1cm/view?usp=share_link

**Comment by aggelos2000430** [comment]
Συνάδελφε χαίρετε, για την τρίτη εβδομάδα είναι αρκετή η εγκατάσταση ενός χειροποίητου συστήματος no-systemd και οι λόγοι που το επέλεξες.
